File: Java/isPalindrome.java
class Solution {

    public boolean isPalindrome(int x) {

        String strX = x + "";
        String reverseStringX = "";

        // Reverse for to strX
        for (int i = strX.length() -1; i >= 0; i-- ) {
            reverseStringX += strX.charAt(i);
        }

        if (strX.equals(reverseStringX)) {
            return true;
        } else {
            return false;
        }

        
    }
}

File: C#/isPalindrome.cs
class Solution  {
    public bool IsPalindrome(int x) {

        String strX = x.ToString();
        String strXReverse = "";

        for(int i = strX.Length-1; i >= 0; i--) {
            strXReverse += strX[i];
        }

        if (strX.Equals(strXReverse)) {
            return true;
        } else {
            return false;
        }
    }
}